src/index.html

```diff
@@ -1,15 +1,205 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+<html lang = "en">
+  <head>
+    <meta charset = "UTF-8">
+    <meta name = "viewport"
+          content = "width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv = "X-UA-Compatible" content = "ie=edge">
+    <title>HTML Form</title>
+    <link rel = "stylesheet" href = "./style.css">
+  </head>
+
+  <body>
+    <form action = "https://mate-academy-form-lesson.herokuapp.com/create-application"
+          onsubmit = "onSubmit()"
+    >
+      <fieldset class = "form-sector">
+        <legend>Personal information</legend>
+
+        <div class = "input-row">
+          <label for = "surname">Surname:</label>
+          <input type = "text"
+                 id = "surname"
+                 name = "surname"
+                 autocomplete = "off"
+                 minlength = "2"
+                 required
+          >
```

Comment:
```suggestion
          <input 
            type = "text"
            id = "surname"
            name = "surname"
            autocomplete = "off"
            minlength = "2"
            required
          >
```
починаючи з першого атрибута потрбіно переносити на наступний рядок і відступ повинен бути тільки на 2 пробіли більшим за той який у тега

Reply:
![image](https://user-images.githubusercontent.com/73616736/98812664-8d6f3000-242b-11eb-9f37-1c5a6b09a67b.png)

руководствовался вашим примером [отсюда](https://github.com/mate-academy/layout_html-form/blob/master/src/index.html))
спасибо)

Reply:
це уловка, теж треба було пофіксити